content/en/integrations/vmware_tanzu_application_service.md

```diff
@@ -0,0 +1,163 @@
+cf restage <YOUR_APP>
+```
+
+#### Trace and profile collection
+
+The Datadog Trace Agent (APM) is enabled by default. Learn more about setup for your specific language in [APM setup][12] and [Profiling setup][30].
+
+#### Log collection
+
+{{< site-region region="us3" >}}
+
+Log collection is not supported for this site.
+
+{{< /site-region >}}
+
+{{< site-region region="us,us5,eu,gov" >}}
+
+##### Enable log collection
+
+To start collecting logs from your application in VMware Tanzu Application Service, the Agent contained in the buildpack needs to be activated and log collection enabled.
+
+```shell
+cf set-env <YOUR_APP_NAME> RUN_AGENT true
+cf set-env <YOUR_APP_NAME> DD_LOGS_ENABLED true
+# Disable the Agent core checks to disable system metrics collection
+cf set-env <YOUR_APP_NAME> DD_ENABLE_CHECKS false
+# Redirect Container Stdout/Stderr to a local port so the Agent collects the logs
+cf set-env <YOUR_APP_NAME> STD_LOG_COLLECTION_PORT <PORT>
+# Configure the Agent to collect logs from the wanted port and set the value for source and service
+cf set-env <YOUR_APP_NAME> LOGS_CONFIG '[{"type":"tcp","port":"<PORT>","source":"<SOURCE>","service":"<SERVICE>"}]'
+# restage the application to make it pick up the new environment variable and use the buildpack
+cf restage <YOUR_APP_NAME>
+```
+
+##### Configure log collection
+
+The following table describes the parameters above, and how they can be used to configure log collection:
+
+| Parameter                 | Description                                                                                                                                |
+| ------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| `RUN_AGENT`               | Set to `true` to start the Datadog Agent.                                                                                                  |
+| `DD_LOGS_ENABLED`         | Set to `true` to enable Datadog Agent log collection.                                                                                      |
+| `DD_ENABLE_CHECKS`        | Set to `false` to disable the Agent's system metrics collection through core checks.                                                       |
+| `STD_LOG_COLLECTION_PORT` | Must be used when collecting logs from `stdout`/`stderr`. It redirects the `stdout`/`stderr` stream to the corresponding local port value. |
+| `LOGS_CONFIG`             | Use this option to configure the Agent to listen to a local TCP port and set the value for the `service` and `source` parameters.          |
+
+**Example:**
+
+A Java application named `app01` is running in VMware Tanzu Application Service. The following configuration redirects the container `stdout`/`stderr` to the local port `10514`. It then configures the Agent to collect logs from that port while setting the proper value for `service` and `source`:
```

Comment:
```suggestion
For example, a Java application named `app01` is running in VMware Tanzu Application Service. The following configuration redirects the container `stdout` or `stderr` to the local port `10514`. It then configures the Agent to collect logs from that port while setting the proper value for `service` and `source`:
```

Reply:
Using `For example` here seems redundant based on the `**Example:**` on line 90.